### PR TITLE
Added bulk release for updateinfo records, code refactoring

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

**Review by maccelf — APPROVED** [review]

File: alws/crud/errata.py [hunk 1]
```diff
@@ -1109,44 +1087,207 @@ async def release_errata_record(db: AsyncSession, record_id: str):
...
+            search_params,
+            pulp,
+            for_release=True,
+        )
+        try:
+            repo_mapping = get_albs_packages_from_record(db_record, pulp_packages)
+        except Exception as exc:
+            db_record.release_status = ErrataReleaseStatus.FAILED
+            db_record.last_release_log = str(exc)
+            logging.exception("Cannot release %s record:", record_id)
```
Comment (javihernandez) [inline alws/crud/errata.py:1204]: Not sure but, shall we log the error too? (in addition to store it as `last_release_log`)
Don't have a strong opinion, but if you ask me, I'd say yes, but I'd be okay if you don't think it's worth doing it ;)

Comment (javihernandez) [reply alws/crud/errata.py:1204]: Nevermind, it's an exception and we'll somehow see what's going on behind the curtains

File: alws/crud/errata.py [hunk 2]
```diff
@@ -1109,44 +1087,207 @@ async def release_errata_record(db: AsyncSession, record_id: str):
...
+            repos_to_publish.extend(repo_mapping.keys())
+            release_tasks.extend(tasks)
+        await session.commit()
+    logging.info("Executing release tasks")
+    await asyncio.gather(*release_tasks)
+    logging.info("Executing publication tasks")
+    await asyncio.gather(
+        *(pulp.create_rpm_publication(href) for href in set(repos_to_publish))
+    )
+    logging.info("Bulk release is finished")
```
Comment (javihernandez) [inline alws/crud/errata.py:1293]: Maybe `Bulk errata release is finished`

**Review by javihernandez — APPROVED** [review]
Nice one!

**Review by javihernandez — APPROVED** [review]

**Review by Korulag — APPROVED** [review]

**Pull request merged**